### DISCUSSION TITLE
Is there any way to keep names of output files as same ?

### DISCUSSION THREAD

**Discussion by abdyek** [parcel-bundler/parcel#8016] [Q&A] [answered]
Hello everyone,

I am using Parcel to build Vue SFC to embed another project. Parcel generates js, html and css files in dist directory for example `index.83145ee1.js`, `index.cd99c106.css`, `index.html`,  cool.. But  therewithal Parcel changes file names each time. These SFCs are piece of a parent project and I am developing parent and its child projects integrately. If I can define a output name for these generated files for example `index.js`, `index.css`, `index.html` then parcel be improved my productivity.

**Comment by CWSites** [comment]
Absolutely, I understand the need for cache busting but it would be GREAT if there was a way to disable this.

**Comment by devongovett** [comment, marked as answer]
Two ways to disable content hashing:

1. Use the `--no-content-hash` CLI option. The file names will still include a hash, but it will be consistent across builds. See [here](https://parceljs.org/features/production/#content-hashing).
2. Use a different [namer plugin](https://parceljs.org/features/plugins/#namers) from the default. For example, [parcel-namer-rewrite](https://www.npmjs.com/package/parcel-namer-rewrite) uses a config to control naming. Find more namer plugins [here](https://parceljs.org/plugin-browser/?type=%22namer%22&page=0&filter=%22%22&includeOfficial=false), or [build your own](https://parceljs.org/plugin-system/namer/) if you need full programmatic control.

**Reply by abdyek** [reply]
Thanks, second way looks suitable for my development environment. I will look at its links.
